ingestion/src/metadata/ingestion/source/database/dbt/metadata.py

```diff
@@ -578,6 +617,44 @@ def parse_data_model_columns(
 
         return columns
 
+    def parse_glossaries_columns(self, manifest_columns: dict) -> Optional[dict]:
+        """
+        Method to parse the DBT columns
+        """
+        try:
+            columns_glossary_list = []
+            for key, manifest_column in manifest_columns.items():
+                try:
+                    dbt_column_meta = DbtMeta(**manifest_column.meta)
+                    logger.debug(f"Processing DBT column glossary: {key}")
+                    if (
+                        dbt_column_meta.openmetadata
+                        and dbt_column_meta.openmetadata.glossary
+                    ):
+                        glossary_labels = get_glossary_labels(
+                            metadata=self.metadata,
+                            glossaries=dbt_column_meta.openmetadata.glossary,
+                            include_tags=self.source_config.includeTags,
+                        )
+                        if glossary_labels:
+                            columns_glossary_list.append(
+                                ColumnGlossary(
+                                    name=manifest_column.name,
+                                    glossary=glossary_labels,
+                                )
+                            )
+                except Exception as exc:  # pylint: disable=broad-except
+                    logger.debug(traceback.format_exc())
+                    logger.warning(f"Failed to parse DBT column glossary {key}: {exc}")
+
+            return columns_glossary_list or None
+
+        except Exception as exc:  # pylint: disable=broad-except
+            logger.debug(traceback.format_exc())
+            logger.warning(f"Failed to parse DBT column glossary : {exc}")
```

Comment:
should this yield an `StackTraceError` as well?

Reply:
Here, it's parsing/getting the glossary information passed on the `manifest.json` file from the `dbt` and returns the list of column `glossary terms` to the `yield_data_models` in the source and its processed there;  also its not a `generator` method. so it doesn't yield `StackTraceError`. Please correct me if i'm thinking something wrong here.